model/item.rb

```diff
@@ -0,0 +1,33 @@
+require 'date'
+class Item
+  attr_reader :id, :archived
+  attr_accessor :genre, :author, :label, :publish_date
+
+  def initialize(publish_date:)
+    @publish_date = Date.parse(publish_date)
+    @id = Random.rand(1..1000)
+    @archived = false
+  end
+
+  def set_genre(genre:)
+    @genre = genre
+  end
+
+  def set_author(author:)
+    @author = author
+  end
+
+  def set_label(label:)
+    @label = label
+  end
```

Comment:
- [ ] the one to many relationship should include the following line.
```
genre.items.push(self) unless genre.items.include?(self)
```
the same goes for Author and Label